repl/src/test/scala/com/cloudera/livy/repl/PythonInterpreterSpec.scala

```diff
@@ -150,6 +149,7 @@ class PythonInterpreterSpec extends BaseInterpreterSpec {
       "name 'x' is not defined",
       List(
         "Traceback (most recent call last):\n",
+        "  File \"<stdin>\", line 1, in <module>\n",
```

Comment:
The stack trace is useful when users are using some complicated library (Pandas) and it returns an error.